core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLSnapshotTestRun.java

```diff
@@ -195,6 +259,21 @@ private void testSource(String sourceETLPlugin, String sourceName, Map<String, I
     Assert.assertEquals(expected, actual);
   }
 
+  private List<String> readTextOutput(Location outputLocation) throws IOException {
```

Comment:
you probably want a Set here, since there is no guarantee on which lines are written to which files.